geometry_msgs/TransformStamped[] footTransforms
---
geometry_msgs/TransformStamped[] jointStates
